src/pages/Login/Login.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import styled from 'styled-components';
+import { KAKAO_AUTH_URL } from './KakaoLogin/KakaoAuth';
+
+const Login = () => {
+  return (
+    <LoginLayout>
+      <LoginTitleWrap>
+        <LoginTitle>LOGIN</LoginTitle>
+        <LoginSub>로그인</LoginSub>
+      </LoginTitleWrap>
+      <LoginForm>
+        <InputWrap>
+          <LoginInput placeholder="이메일 아이디" />
+          <LoginInput placeholder="비밀번호" />
+        </InputWrap>
+        <LoginBtn login onClick={e => e.preventDefault()}>
+          LOGIN
+        </LoginBtn>
+        <LoginBtn search onClick={e => e.preventDefault()}>
+          비회원 예약조회
+        </LoginBtn>
+        <SignupWrap>
+          <h3>회원가입</h3>
+          <h3>비밀번호 찾기</h3>
+        </SignupWrap>
+      </LoginForm>
+      <LoginSocialWrap>
+        <LoginSocialTitle>SNS 계정으로 로그인하기</LoginSocialTitle>
+        <LoginList>
+          <LoginItems>
+            <KakaoBtnImg
+              src="https://www.stayfolio.com/web/images/sns_naver.png"
+              alt="loginIcon"
+            />
+          </LoginItems>
+          <LoginItems>
+            <KaKaoLoginBtn href={KAKAO_AUTH_URL}>
+              <KakaoBtnImg
+                src="https://www.stayfolio.com/web/images/sns_kakao.png"
+                alt="loginIcon"
+              />
+            </KaKaoLoginBtn>
+          </LoginItems>
+          <LoginItems>
+            <KakaoBtnImg
+              src="https://www.stayfolio.com/web/images/sns_fb.png"
+              alt="loginIcon"
+            />
+          </LoginItems>
+          <LoginItems>
+            <KakaoBtnImg
+              src="https://www.stayfolio.com/web/images/sns_apple.png"
+              alt="loginIcon"
+            />
+          </LoginItems>
+        </LoginList>
+      </LoginSocialWrap>
+    </LoginLayout>
+  );
+};
+
+export default Login;
+
+const LoginLayout = styled.section`
+  margin: 0 auto;
+  min-height: 800px;
+`;
+
+const LoginTitleWrap = styled.div`
+  width: 100%;
+  height: 206px;
+  padding-top: 70px;
+  text-align: center;
+`;
+
+const LoginTitle = styled.h1`
+  font-weight: 700;
+  font-size: 18px;
+  letter-spacing: 13px;
+  text-indent: 13px;
+`;
+
+const LoginSub = styled.h3`
+  margin-top: 15px;
+  font-size: 15px;
+`;
+
+const InputWrap = styled.div`
+  margin: 18px 0 25px;
+`;
+
+const LoginForm = styled.form`
+  margin: 0 auto;
+  width: 360px;
+  border-top: 1px solid #000000;
+`;
+
+const LoginInput = styled.input`
+  width: 100%;
+  height: 45px;
+  margin-top: 13px;
+  font-size: 14px;
+  border: none;
+  border-bottom: 1px solid #dddddd;
+  outline: none;
+  &::placeholder {
+    color: #bbbbbb;
+  }
+`;
+
+const LoginBtn = styled.button`
+  width: 100%;
+  height: ${props => (props.login ? '55px' : '45px')};
```

Comment:
```suggestion
  height: ${{login} => (login ? '55px' : '45px')};
```
이렇게 구조분해할당 해줄 수 있습니다

Reply:
구조분해 할당 하였습니다!